cli/src/main/kotlin/commands/ReporterCommand.kt

```diff
@@ -100,7 +100,7 @@ class ReporterCommand : CliktCommand(
 
     private val copyrightGarbageFile by option(
         "--copyright-garbage-file",
-        help = "A file containing garbage copyright statements entries which are to be ignored."
+        help = "A file containing copyright statements which are marked as garbage."
```

Comment:
Commit message: consider saying "Whether copyright statements....",  instead of "If statements"

Reply:
I have reworded the whole commit message.